src/components/Router/StepComponentMap.ts

```diff
@@ -171,17 +176,24 @@ const buildFaceComponents = (
   return shouldUseVideo
     ? buildRequiredVideoComponents(
         deviceHasCameraSupport && videoCameraSupport,
-        mobileFlow
+        mobileFlow,
+        showClientSessionLinkedIntro
+      )
+    : buildRequiredSelfieComponents(
+        shouldSelfieScreenUseCamera,
+        showClientSessionLinkedIntro
       )
-    : buildRequiredSelfieComponents(shouldSelfieScreenUseCamera)
 }
 
 const buildRequiredVideoComponents = (
   shouldUseCamera?: boolean,
-  mobileFlow?: boolean
+  mobileFlow?: boolean,
+  showClientSessionLinkedIntro?: boolean
 ): ComponentType<StepComponentProps>[] => {
-  // @TODO: convert FaceVideoIntro, FaceVideoCapture, FaceVideoConfirm to TS
-  const allVideoSteps = [FaceVideoIntro, FaceVideoCapture, FaceVideoConfirm]
+  // @TODO: convert FaceVideoCapture, FaceVideoConfirm to TS
+  const allVideoSteps = showClientSessionLinkedIntro
+    ? [FaceVideoIntro, ClientSessionLinked, FaceVideoCapture, FaceVideoConfirm]
```

Comment:
I would challenge the decision of displaying this screen after the Selfie/Video Intro screen. I think it should be the first screen that a user sees when they land on the `mobileFlow`. I think that a user might be inclined to skip the information on this screen if they are already getting ready for the capture.

Reply:
I can rename this to make it clearer, but the screen is only displayed when they first land on the `mobileFlow` Cross Device client. Should be clearer also with the other proposed change to address your earlier feedback. 
Was trying to avoid complexity or even having to make the change here in general but this seemed like the best/least complex place to implement it. Agree that this does add further to the complexity here. Open to suggestions if you can see a better way to add this screen into the flow.

Reply:
I see, thanks for the clarification. In that case, there is probably some logic I'm not familiar with in the Router that handles it. Might be good to add a comment or refactor because it's not obvious from the stepComponentMap that ClientSessionLinked is the first screen a user would see on the cross device client.